### PR TITLE
WWST-1615 Add fake battery event and different metadata for Ozom CO

### PR DESCRIPTION
**Pull request by dkirker**

### CODE REVIEW

**Comment by dkirker** [comment]
So, I had thought the Ozom fingerprint was already added. Apparently, that PR was closed. Is there a reason the fingerprint in https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/3717 wasn't just put in a DTH for CO? I am considering moving it to something like that.

**Review by tpmanley — DISMISSED** [review]

**Comment by greens** [comment]
Did they end up in the Orvibo Gas Detector DTH which is the same as this one sans battery?

**Comment by dkirker** [comment]
The last comment I see on #3717 is that #3717 was closed waiting for HCS-299. I'll go check. I commented on the ticket, as well, so if there are no objections I am going to move this to a proper Zigbee CO DTH.

**Comment by dkirker** [comment]
@greens No. As far as I can tell the fingerprint is not in a merged PR. I just created a new DTH so that it uses the correct capabilities.

File: devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy
Comment (greens) [inline devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy]: needs to be `response(refresh())` I think?

Comment (dkirker) [reply devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy]: Yeah, I think installed, updated, and uninstalled need it... This was copied from the Zigbee Smoke, which leads me to believe that it was never tested...

File: devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy
Comment (greens) [inline devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy]: you can change this too if you want

**Review by greens — DISMISSED** [review]
Copyright date is off

File: devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy
Comment (dkirker) [reply devicetypes/smartthings/zigbee-co-sensor.src/zigbee-co-sensor.groovy]: Eh. The file I copied 98% of this from is 2018. 🤷‍♂️

**Comment by dkirker** [comment]
Bumped the copyright date. @greens since this is pointed to staging we should probably wait until tomorrow to merge. I could try to rebase, but I am sure it would just get ugly.

**Review by greens — APPROVED** [review]
Merge whenever.

**Comment by workingmonk** [comment]
bumping up copyright is actually a bad practice. older copyright shows that we created the file in 2018 and have the copyright since then. we could leave it as is or do `2016-2019`

**Comment by greens** [comment]
@workingmonk It's a new DTH.

**Pull request merged**
